traffic_portal/app/src/common/modules/table/deliveryServiceStaticDnsEntries/TableDeliveryServiceStaticDnsEntriesController.js

```diff
@@ -27,6 +27,10 @@ var TableDeliveryServiceStaticDnsEntriesController = function(deliveryService, s
 		$state.reload(); // reloads all the resolves for the view
 	};
 
+    $scope.editStaticDnsEntry = function(dsId, staticDnsEntryId) {
+        locationUtils.navigateToPath('/delivery-services/' + dsId + '/static-dns-entry/' + staticDnsEntryId);
```

Comment:
you should make this plural to conform with the rest of TP's routes. so like this

+        locationUtils.navigateToPath('/delivery-services/' + dsId + '/static-dns-entries/' + staticDnsEntryId);